srml/treasury/src/lib.rs

```diff
@@ -141,6 +159,12 @@ decl_module! {
 		}
 
 		/// Reject a proposed spend. The original deposit will be slashed.
+		///
+		/// # <weight>
+		/// - proposal_id is a u32
+		/// - possible complexity in `on_unbalanced` *
```

Comment:
Why does this end with a * ?
```suggestion
		/// - Possible complexity in `on_unbalanced` *
```

Reply:
@lsaether are usually implemented via an input config type in the `pub trait Trait { ... }` and implemented in `runtime/src/lib`. atm they are mostly nothing `()`.

Reply:
In that case I can remove that comment line.